_format/1.1/index.md

```diff
@@ -1232,6 +1232,65 @@ Accept: application/vnd.api+json
 A server **MUST** return `403 Forbidden` in response to an unsupported request
 to create a resource with a client-generated ID.
 
+#### <a href="#crud-sideposting" id="crud-sideposting" class="headerlink"></a> Sideposting
+
+A server **MAY** accept a request to create related resources along with the primary
+resource, which is referred to as *sideposting*. A related resource **MUST**
+be placed within the `included` member, and have a `temp-id` member the value of which
+is an arbitrary string that uniquely identifies a related resource within the request
+document. The related resources **MUST** be referred to (from the linkage data of
+resource relationships) via a *temporary resource identifier object* (that is an
+object with a single `temp-id` member referring to a related resource).
+
+The document **MUST** still respect the full linkage requirement.
+
+For instance, a new article might be created along with two tags with the following 
+request:
+
+```http
+POST /articles HTTP/1.1
+Content-Type: application/vnd.api+json
+Accept: application/vnd.api+json
+
+{
+  "data": {
+    "type": "articles",
+    "attributes": {
+      "title": "Sideposting with json:api"
+    },
+    "relationships": {
+      "tags": {
+        "data": [{ "type": "tags", "id": "9" },
+                 { "temp-id": "1" },
```

Comment:
The idea is that temp-ids identify a unique resource within the document, so the type is redundant. I agree that this introduces a difference with the "normal" resource identifiers, which only require the id to be unique *per type*. I do not feel strongly about it, so let's wait for others to chip in before making a call on that.